dali/operators/image/crop/crop_mirror_normalize.cc

```diff
@@ -34,6 +34,9 @@ Normalization takes the input images and produces the output by using the follow
 )code")
   .NumInput(1)
   .NumOutput(1)
+  .InputLayout(0, {"HWC", "CHW", "DHWC",
+                   "FHWC", "FCHW", "CDHW", "CFHW",
```

Comment:
Nitpick: I know it technically only matters which layout is listed _first_ for any given number of dimensions, but it would be nice to group them as 3D, video (or, conversely: video, 3D).
```suggestion
  .InputLayout(0, {"HWC", "CHW", "DHWC", "CDHW",
                   "FHWC", "FCHW","CFHW",
```